whylog_vim/tests/tests_proxy/tests_unit_proxy_main.py

```diff
@@ -1,25 +1,31 @@
 from unittest import TestCase
-from mock import MagicMock
+from mock import MagicMock, patch
+
+from whylog import FrontInput, LineSource
 
 from whylog_vim.consts import EditorStates as States
 from whylog_vim.proxy import WhylogProxy
 
 
 class UnitTestWhylogProxy(TestCase):
-    def tests_unit_check_log_reader_states_of_whylog_proxy(self):
-        whylog_proxy = WhylogProxy(MagicMock())
+    def setUp(self):
+        mock_editor = MagicMock()
+        mock_editor.get_front_input.return_value = FrontInput(
+                1, 'line content', LineSource('host', 'path'))  # yapf:disable
```

Comment:
don't disable yapf here and even when you do disable it, format the line properly before disabling it